code/src/components/SurveyForm.js

```diff
@@ -0,0 +1,63 @@
+import React, {useState} from 'react'
+import Start from './Start.js'
+import Age from './Age.js'
+import Purpose from './Purpose.js'
+import Expectations from './Expectations.js'
+import Summary from './Summary.js'
+
+const SurveyForm = () => {
+const [ageGroup, setAgeGroup] = useState('')
+const [options, setOptions] = useState ('choose below')
+const [expectations, setExpectations] = useState('')
+const [slide, setSlide] = useState(1)
+
+const nextSlide = () => {
+    setSlide(slide + 1)
+}
+const backSlide = () => {
+    setSlide(slide - 1)
+}
+//Rendering all components 
+    return (
+      <div>
+        <form>
```

Comment:
Nice approach with using the `<form>` tag here. In a semantic approach I am interested to know if  it's the correct way to use <form> since you have the `<label>` and `<input>` tags in your components.